bitnami/influxdb/templates/application.yaml

```diff
@@ -0,0 +1,323 @@
+            - name: GOOGLE_SERVICE_ACCOUNT_FILE
+              value: "/opt/bitnami/influxdb/secrets/google-service-account-key"
+            {{- else }}
+            - name: GOOGLE_SERVICE_ACCOUNT
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "influxdb.store.secret.name" . }}
+                  key: google-service-account-key
+            {{- end }}
+            {{- else if eq .Values.objectStore "azure" }}
+            - name: AZURE_STORAGE_ACCOUNT
+              value: {{ .Values.azure.account | quote }}
+            {{- if .Values.usePasswordFiles }}
+            - name: AZURE_STORAGE_ACCESS_KEY_FILE
+              value: "/opt/bitnami/influxdb/secrets/azure-storage-access-key"
+            {{- else }}
+            - name: AZURE_STORAGE_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "influxdb.store.secret.name" . }}
+                  key: azure-storage-access-key
+            {{- end }}
+            {{- end }}
+            {{- if .Values.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" .) | nindent 12 }}
+            {{- end }}
+          {{- if or .Values.extraEnvVarsCM .Values.extraEnvVarsSecret }}
+          envFrom:
+            {{- if .Values.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ .Values.extraEnvVarsCM }}
+            {{- end }}
+            {{- if .Values.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ .Values.extraEnvVarsSecret }}
+            {{- end }}
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.containerPorts.http }}
+              protocol: TCP
+            {{- if .Values.extraContainerPorts }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.extraContainerPorts "context" .) | nindent 12 }}
+            {{- end }}
+          {{- if not .Values.diagnosticMode.enabled }}
+          {{- if .Values.customStartupProbe }}
+          startupProbe: {{- include "common.tplvalues.render" (dict "value" .Values.customStartupProbe "context" .) | nindent 12 }}
+          {{- else if .Values.startupProbe.enabled }}
+          {{- $startupTimeout := sub (int .Values.startupProbe.timeoutSeconds) 1 }}
+          startupProbe: {{- omit .Values.startupProbe "enabled" | toYaml | nindent 12 }}
+            tcpSocket:
+              port: http
+          {{- end }}
+          {{- if .Values.customLivenessProbe }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.customLivenessProbe "context" .) | nindent 12 }}
+          {{- else if .Values.livenessProbe.enabled }}
+          {{- $livenessTimeout := sub (int .Values.livenessProbe.timeoutSeconds) 1 }}
+          livenessProbe: {{- omit .Values.livenessProbe "enabled" | toYaml | nindent 12 }}
+            tcpSocket:
```

Comment:
There is a `ping` endpoint, not sure if it can be used instead of tcpSocket (which depending on the app it adds noise to the logs)

https://github.com/influxdata/influxdb/blob/bf83e7fbb397e579cbee0ca87c5b797fc2ecdcf5/influxdb3_server/src/all_paths.rs#L25

Reply:
It can't be used given Bearer Token Auth is required for every endpoint (including "ping"), see https://docs.influxdata.com/influxdb3/core/api/v3/#operation/GetPing